types.go

```diff
@@ -11,11 +11,6 @@ import (
 // that will be initialized in the kernel.
 type MapType uint32
 
-// Max returns the latest supported MapType.
-func (MapType) Max() MapType {
-	return maxMapType - 1
```

Comment:
maxMapType/maxProgramType can also be removed. They're still referred to by Stringer-generated files, so they're not flagged by unused.

Reply:
Thanks!